setup/fusion/scripts/Comp/colorbleed/switch_ui.py

```diff
@@ -154,8 +171,12 @@ def _on_switch(self):
 
         asset = self._assets.currentText()
 
-        import colorbleed.scripts.fusion_switch_shot as switch_shot
-        switch_shot.switch(asset_name=asset, filepath=file_name, new=True)
+        # Get  switch and submit module
+        script_dir = _get_script_dir()
+        site.addsitedir(script_dir)
+
+        import switch_and_submit as switch_shot
```

Comment:
Importing a script like this seems hacky, kind of feels like the `switch_shot` method is almost a library function of sorts or should just be embedded here in this file. Not sure if there's any use case outside of this file?

It would be much easier if we could just make this file easier accessible then go through all this length to find it and import it. Can't we simplify this?